README.md

```diff
@@ -1,22 +1,20 @@
 # ROS Gremsy Gimbal Interface
-A ROS interface to control Gremsy gimbals. Based on the [gSDK](https://github.com/Gremsy/gSDK) interface and the MavLink protocol.
+A ROS interface to control Gremsy V3 gimbals. Based on the [gSDK_V3_alpha](https://github.com/Gremsy/gSDK/tree/gSDK_V3_alpha) interface and the MavLink protocol.
 
 Disclaimer: This software package is not officially developed by or related to Gremsy.
 
-![Build CI](https://github.com/Flova/ros_gremsy/workflows/Build%20CI/badge.svg)
-
 ## Description
-This package includes a ROS Node which warps the gSDK for the Gremsy Gimbals which are mainly used for physical image stabilization.
+This package includes a ROS Node which warps the gSDK for the Gremsy V3 gimbals which are mainly used for physical image stabilization.
 
 The gimbal is connected via UART with a Linux host device running this node.
 Devices such as the Raspberry Pi feature a build-in UART interface others like most PCs or Laptops need a cheap USB Adapter.
 The used serial device, as well as many other gimbal specific parameters, can be configured in the `config.yaml` file.
-The node publishes the gimbals encoder positions, imu measurements, and the camera mount orientation.
+The node publishes the gimbals encoder positions.
```

Comment:
Is the IMU, etc not supported by the SDK or is it not implemented in the ROS interface?

Reply:
I've only been able to implement the publication of the gimbal encoders with the new SDK. In the case of the IMU, the method used to get the data returns without closing a mutex and the system blocks. I think that it's a bug in the SDK but  might be due to an incorrect use of the method. Regarding the camera mount orientation, I don't find a method in the new SDK to read the data. However, I haven't studied the functionalities of the new interface thoroughly so I could have a look to ensure they can't be implemented.